sdks/python/apache_beam/io/components/adaptive_throttler.py

```diff
@@ -94,3 +117,49 @@ def successful_request(self, now):
       now: int, time in ms since the epoch
     """
     self._successful_requests.add(now, 1)
+
+
+class ReactiveThrottler(AdaptiveThrottler):
+  """ A wrapper around the AdaptiveThrottler that also handles logging and
+  signaling throttling to the SDK harness.
```

Comment:
Could we include example usage here and be a little more descriptive on what this does?

I think we want people to use this instead of AdaptiveThrottler, so it would be better to be explicit about the UX

Reply:
Also, could we update CHANGES.md?

Reply:
updated CHANGE.md + added a pseudocode workflow for users in the docstring